src/ucs/config/parser.c

```diff
@@ -366,7 +366,7 @@ ucs_status_t ucs_config_clone_log_comp(const void *src, void *dst, const void *a
 
 int ucs_config_sscanf_bitmap(const char *buf, void *dest, const void *arg)
 {
-    char *str = strdup(buf);
+    char *str = ucs_strdup(buf, "config_sscanf_bitmap str");
```

Comment:
can we use ucs_alloca() here?

Reply:
technically yes, but value may come from external configuration and may contain few MB of data (@brminich seen issue where environment variable was filled by debug output) and process may crash due to stack overflow